Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorHollowEffect.py

```diff
@@ -110,14 +110,14 @@ def updateGUIFromMRML(self):
       selectedSegmentLabelmapSpacing = selectedSegmentLabelmap.GetSpacing()
       shellThicknessPixel = self.getShellThicknessPixel()
       if shellThicknessPixel[0] < 1 or shellThicknessPixel[1] < 1 or shellThicknessPixel[2] < 1:
-        self.shellThicknessPixel.text = "Not feasible at current resolution."
+        self.shellThicknessLabel.text = "Not feasible at current resolution."
         self.applyButton.setEnabled(False)
       else:
         thicknessMm = self.getShellThicknessMm()
-        self.shellThicknessPixel.text = "Actual: {0} x {1} x {2} mm.".format(*thicknessMm)
+        self.shellThicknessLabel.text = "Actual: {0} x {1} x {2} mm ({3}x{4}x{5} pixels)".format(*thicknessMm, *shellThicknessPixel)
```

Comment:
Assuming `pixel` is a unit, it should probably be in singular form. See https://physics.stackexchange.com/a/384189

@pieper  What do you think ?

Reply:
Yes, as used here it would make sense to use the singular "pixel" when used as a unit.

Also, if we want to stick with the style guide, it would be `thicknessMM` (although I'll admit it looks just as odd as `thicknessMm` will probably annoy the mathematician/physicists just as much).
https://www.slicer.org/wiki/Documentation/Nightly/Developers/Style_Guide#Naming